libs/s25main/desktops/dskOptions.cpp

```diff
@@ -499,6 +508,13 @@ void dskOptions::Msg_Group_ComboSelectItem(const unsigned group_id, const unsign
             VIDEODRIVER.setTargetFramerate(SETTINGS.video.framerate);
             break;
         case ID_cbVideoDriver: SETTINGS.driver.video = combo->GetText(selection); break;
+        case ID_cbGuiScale:
+            if(selection == 0)
+                SETTINGS.video.guiScale = 0;
+            else
+                SETTINGS.video.guiScale = guiScales_[selection - 1];
```

Comment:
To avoid the special case and confusing difference between "selection" and "index", I'd suggest to add a zero entry at the beginning of `guiScales` and then:

```suggestion
            SETTINGS.video.guiScale = guiScales_[selection];
```